recipes/pyearth/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "pyearth" %}
+{% set name_pypi = "sklearn-contrib-py-earth" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "3d0f1efa5f5508610500deec0fe1084716acce1d0fc4fc81d48c52791ce7ba0c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name_pypi }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name_pypi[0] }}/{{ name_pypi }}/{{ name_pypi }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - toolchain 
+    # Notes:
+    # 1. Cython is not used in build as
+    #     a) .c files are included
+    #     b) with the current setup.py cythonizing requires an additional
+    #        parameter
+    # 2. pyearth cimports BLAS from scipy
+    #    These combinations of pinned numpy/scipy versions
+    #    are known to work, see related discussion in
+    #    conda-forge/staged-recipes#4808 and conda-forge/implicit-feedstock#4
+    - numpy 1.11.*
+    - scipy 0.18.*
+    - six
+
+  run:
+    - python
+    - six
+    - numpy >=1.11
+    - scipy >=0.18
+    - scikit-learn >=0.16
+
+test:
+  imports:
+    - pyearth
```

Comment:
Would it be possible to add a few more import tests?

Reply:
Thanks for the review @CJ-Wright !

Well, `__init__.py` imports `earth.py` that imports all the Cython packages (at least no other Python file does) so that's why I though it might be sufficient.

Reply:
Ah ok.